src/operations/channels.jl

```diff
@@ -0,0 +1,323 @@
+"""
+    SplitChannels <: Augmentor.Operation
+
+Description
+--------------
+
+Splits out the color channels of the given image using the
+function `ImageCore.channelview`. This will effectively create a
+new array dimension for the colors in the front. In contrast to
+`ImageCore.channelview` it will also result in a new dimension
+for Gray images.
```

Comment:
Well I suspect the reason this is common in deep learning is simply so that color images and greyscale images can use the same underlying code and GPU kernels.

Personally I prefer the `channelview` behaviour, but the goal of this package is to serve as a convenient glue between the image ecosystem and deep learning